### DISCUSSION TITLE
safari脚本用不了

### DISCUSSION THREAD

**Discussion by jinbeifenghou** [the1812/Bilibili-Evolved#4249] [提问 / Q&A]
我有两台Mac，系统版本是一样的，Safari版本也都是16.5，但是24寸的iMac可以正常使用，14寸的MacBook pro使用后没有反应，屏幕左侧没有出现按钮，但是顶栏不见了，换源之后也一样，求一个大佬解答

**Comment by jinbeifenghou** [comment]
刚刚又试了一下，发现adguard插件打开时左侧的设置按钮就能出现，可以安装插件，但是网页刷新后安装的功能就全部清空了，顶栏还是不显示；adguard关闭后左侧的设置按钮就不显示了。

**Reply by the1812** [reply]
怎么听着像是把脚本装到 AdGuard 里了

**Reply by jinbeifenghou** [reply]
没有，我同时安装了油猴和adguard，脚本装在油猴里面，adguard里啥脚本都没装，只是adguard不知道为什么对脚本的设置按钮有封印效果
